lib/managers/files.js

```diff
@@ -576,6 +576,29 @@ Files.prototype.deletePermanently = function(fileID, callback) {
 };
 
 /**
+ * Used to retrieve an expiring URL for creating an embedded preview session.
+ * The URL will expire after 60 seconds and the preview session will expire after 60 minutes.
+ *
+ * API Endpoint: '/files/:fileID?fields=expiring_embed_link'
+ * Method: GET
+ *
+ * @param {string} fileID - The ID of the file to generate embed link for
+ * @param {Function} callback - Called with nothing if successful, error otherwise
```

Comment:
This comment needs to be updated; the callback will be called with the embed link when successful